### PR TITLE
feat(lang): Japanese translation

### PR DESCRIPTION
**Pull request by Fyphen1223**
Added Japanese translation.

### CODE REVIEW

File: lang/ja.json [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+                "noRole": "ミュートロールがセットされていません。`{prefix}setmute`を使用して設定してください。",
+                "slashMemberDescription": "誰をミュートしたいですか？",
+                "slashReasonDescription": "ミュートの理由",
+                "usage": "{prefix}mute <@メンション | ID> [理由]",
+                "userMuted": "あなたは**{guildName}**で**`ミュート`**されました。",
+                "userNoPermission": "あなたに**`MANAGE ROLES`**の権限がありません。"
+            },
+            "purge": {
+                "botNoPermission": "**`MANAGE MESSAGES`**の権限がありません。",
+                "description": "メッセージを削除"
```
Comment (Mednoob) [inline lang/ja.json]: You forgot to add a comma here. The language can't load if the JSON is invalid.

**Review by Mednoob — CHANGES_REQUESTED** [review]
You need to add the language code to [`src/config/index.ts`](https://github.com/stegripe/rawon/blob/main/src/config/index.ts#L43) and [`.env_example`](https://github.com/stegripe/rawon/blob/main/.env_example#L36). Also, add your github to the [`README file`](https://github.com/stegripe/rawon/blob/main/README.md?plain=1#L105).

You can click the links above to see the lines where you need to change/add stuff.

File: lang/ja.json (hunk 1 above)
Comment (Fyphen1223) [reply lang/ja.json]: Sorry, I completely forgot that lmao

**Comment by Fyphen1223** [comment]
@Mednoob Thanks, I think I did all things that you mentinoed.

**Review by Mednoob — APPROVED** [review]
LGTM! 🎉

Thank you for the translation 👍

**Pull request merged**